src/main/java/smartthings/ratpack/sqs/DefaultSqsService.java

```diff
@@ -35,7 +37,8 @@ public DefaultSqsService(
     @Override
     public void onStart(StartEvent event) throws Exception {
         if (config.isEnabled()) {
-            this.sqs = sqsConfigService.getAmazonSQSAsync(config.getRegion());
+            sqs = sqsConfigService.getAmazonSQSAsync(config.getRegion());
+            Optional.ofNullable(config.getEndpoint()).ifPresent(sqs::setEndpoint);
```

Comment:
Also, @timic this library never really got out of the prototype stage.  So there are likely other issues.  I started it, but my use-case disappeared.